sdk/monitor/Microsoft.Azure.Management.Monitor/src/Generated/ActionGroupsOperations.cs

```diff
@@ -920,10 +920,57 @@ public async Task<AzureOperationResponse<ActionGroupResource>> UpdateWithHttpMes
         /// <param name='cancellationToken'>
         /// The cancellation token.
         /// </param>
-        public async Task<AzureOperationResponse<TestNotificationResponse>> PostTestNotificationsWithHttpMessagesAsync(NotificationRequestBody notificationRequest, Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken))
+        public async Task<AzureOperationHeaderResponse<ActionGroupsPostTestNotificationsHeaders>> PostTestNotificationsWithHttpMessagesAsync(NotificationRequestBody notificationRequest, Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken))
```

Comment:
I don't see any change in test, do these updated APIs have corresponding tests? If not, could you add tests to see if these APIs could work as expected.